src/common/tracer.h

```diff
@@ -0,0 +1,67 @@
+#ifndef TRACER_H_
```

Comment:
please split to header and cc file

Reply:
I dont need a .cc file for anything should i just make .cc file and one line of including .h file?

Reply:
you **do** need cc file - this will minimize dependencies.
other files that include tracer.h should not have dependency with jaeger

Reply:
noted